cmd/notary/main.go

```diff
@@ -1,96 +1,71 @@
 package main
 
 import (
-	"crypto/x509"
 	"fmt"
 	"os"
 	"os/user"
-	"path"
-	"time"
+	"path/filepath"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	"github.com/docker/notary/trustmanager"
+	"github.com/docker/notary/pkg/passphrase"
 )
 
 const configFileName string = "config"
-const configPath string = ".docker/trust/"
-const trustDir string = "trusted_certificates/"
-const privDir string = "private/"
-const rootKeysDir string = "root_keys/"
+const defaultTrustDir string = ".notary/"
```

Comment:
I realize that we don't want to tie this tool to docker, but this means that you need to override the default trust dir to use it with docker, right? Isn't that awkward UX?